int32 object_label
float32 size
geometry_msgs/PointStamped pedestrian_laser
